msg/ManualControlSetpoint.msg

```diff
@@ -16,31 +16,15 @@ uint8 data_source
 
 # Any of the channels may not be available and be set to NaN
 # to indicate that it does not contain valid data.
-# The variable names follow the definition of the
-# MANUAL_CONTROL mavlink message.
-# The default range is from -1 to 1 (mavlink message -1000 to 1000)
-# The range for the z variable is defined from 0 to 1. (The z field of
-# the MANUAL_CONTROL mavlink message is defined from -1000 to 1000)
-
-float32 x			 # stick position in x direction -1..1
-				 # in general corresponds to forward/back motion or pitch of vehicle,
-				 # in general a positive value means forward or negative pitch and
-				 # a negative value means backward or positive pitch
-
-float32 y			 # stick position in y direction -1..1
-				 # in general corresponds to right/left motion or roll of vehicle,
-				 # in general a positive value means right or positive roll and
-				 # a negative value means left or negative roll
-
-float32 z			 # throttle stick position 0..1
-				 # in general corresponds to up/down motion or thrust of vehicle,
-				 # in general the value corresponds to the demanded throttle by the user,
-				 # if the input is used for setting the setpoint of a vertical position
-				 # controller any value > 0.5 means up and any value < 0.5 means down
-
-float32 r			 # yaw stick/twist position, -1..1
-				 # in general corresponds to the righthand rotation around the vertical
-				 # (downwards) axis of the vehicle
+
+# Stick positions [-1,1]
+# on a common RC mode 1/2/3/4 remote/joystick the stick deflection: -1 is down/left, 1 is up/right
+# Note: QGC sends throttle/z in range [0,1000] - [0,1]. The MAVLink input conversion [0,1] to [-1,1] is at the moment kept backwards compatible.
+# Positive values are generally used for:
```

Comment:
Previously the mappings were vehicle and flight mode agnostic. An MC vehicle in position mode does not roll or pitch - it move forward or left-right. So you've made this very unintuitive except for fixed wing vehicles.

Reply:
> Previously the mappings were vehicle and flight mode agnostic.

I do not agree, previously the naming according to the cartesian coordinate system's axes `x`, `y`, `z` only fully made sense for multirotor position mode where the values moved the drone along the respective body axis. `r` I could never make sense of.

> An MC vehicle in position mode does not roll or pitch - it move forward or left-right.

I agree it doesn't fit entirely and it's in my eyes impossible to find a naming that fully fits every single possible mode, that's why I favored consistency across drone remotes.

> So you've made this very unintuitive except for fixed wing vehicles.

I disagree. For someone who never worked with drones before and has no reference from any other project or product you could argue the previous naming was equal as suitable. But this is clearly not the case. As I elaborated in https://github.com/PX4/PX4-Autopilot/pull/15949#issuecomment-1168325766 people come with previous knowledge and or use a remote of some brand and the roll, pitch, yaw, throttle naming is the most popular and consistent one out in the hobby and industry. So it's not just me who got confused when joining PX4 why it needs to have different naming than everyone else, every time has to think twice about which axis it now is and keep a translation table in the comments but numerous other people I asked and found reference for in the code.

Reply:
After 5 years working with PX4 I still had to look up every time what x,y,z and r are linked to, so for me these new namings are massively more intuitive.